aea/cli/utils/click_utils.py

```diff
@@ -148,6 +148,16 @@ def wrapper(f):
     return wrapper
 
 
+def registry_path_option(f):
+    """Add registry path aea option."""
+    return option(
+        "--registry-path",
+        type=str,
```

Comment:
Please use `click.Path` to have click validate the path (see other commands)

Reply:
thanks! nice point